src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -744,6 +753,86 @@ def load_ldlc_medication_effect(key: str, location: str) -> pd.DataFrame:
     return df
 
 
+def load_relative_risk_categorical_sbp(key: str, location: str) -> pd.DataFrame:
+    distributions = data_values.RELATIVE_RISK_SBP_ON_HEART_FAILURE_DISTRIBUTIONS
+    population_structure = load_population_structure(
+        data_keys.POPULATION.STRUCTURE, location
+    ).droplevel("location")
+
+    # define TMREL data
+    baseline_hf_rrs = pd.DataFrame(
+        1.0, index=population_structure.index, columns=ARTIFACT_COLUMNS
+    )
+    baseline_hf_rrs["parameter"] = "cat4"
+
+    # define exposed groups data
+    exposed_groups_rrs = []
+    for sbp_category, distribution in distributions:
+        rr_data = get_random_variable_draws(DRAW_COUNT, (sbp_category, distribution))
+        # relative risks of 1 for ages without heart failure (under 25)
+        under_25_data = pd.DataFrame(
+            1,
+            index=population_structure.query("age_start<=25").index,
```

Comment:
Do we include 25.0 here?

Reply:
Yup. I was thinking it might be better to change the name to under_and_including_25 anyway to make it more clear which interval includes that age group.